core/ui/Actions/new-journal.tid

```diff
@@ -2,7 +2,7 @@ title: $:/core/ui/Actions/new-journal
 tags: $:/tags/Actions
 description: create a new journal tiddler
 
-<$vars journalTitleTemplate={{$:/config/NewJournal/Title}} journalTags={{$:/config/NewJournal/Tags}} journalText={{$:/config/NewJournal/Text}}>
+<$vars journalTitleTemplate={{$:/config/NewJournal/Title}} journalTags={{{ [{$:/config/NewJournal/Tags}addsuffix[ ]addsuffix{$:/language/DefaultNewTiddlerTags!!tags}] }}} journalText={{$:/config/NewJournal/Text}}>
```

Comment:
Should new journal tiddlers get these tags? Also it might be good to add a UI for $:/config/NewJournal/Tags

Reply:
I thought it would be good that also the journal tiddlers get those additional tags

Reply:
I'm not sure what's best. Visualising the control panel with a row for editing new tiddler tags and a row for new journal tags I suspect that users would perceive them as independent, and we'd have to include some instructions to explain what's going on. So I guess I'm in favour of keeping them separate. People can always duplicate the tags; it's hard for them to separate them if we leave it like this.

Reply:
It now uses only the separate new-journal tags for new journal tiddlers. the UI in the controlpanel now is the same for both

Reply:
Thanks @BurningTreeC